src/Razor/test/VSCode.FunctionalTest/tests/index.ts

```diff
@@ -27,7 +27,7 @@ export async function run(): Promise<void> {
     }
 
     return new Promise((c, e) => {
-        glob('**/**.test.js', { cwd: testsRoot }, (err, files) => {
+        glob('**/Hover.test.js', { cwd: testsRoot }, (err, files) => {
```

Comment:
I assume this is a temporary thing, keeping turn-around time low by running fewer tests?

Reply:
Yup!